statistics/handle/update.go

```diff
@@ -502,28 +503,62 @@ func (h *Handle) UpdateErrorRate(is infoschema.InfoSchema) {
 
 // HandleUpdateStats update the stats using feedback.
 func (h *Handle) HandleUpdateStats(is infoschema.InfoSchema) error {
-	sql := "select table_id, hist_id, is_index, feedback from mysql.stats_feedback order by table_id, hist_id, is_index"
-	rows, _, err := h.restrictedExec.ExecRestrictedSQL(sql)
-	if len(rows) == 0 || err != nil {
+	sql := "SELECT distinct table_id from mysql.stats_meta"
```

Comment:
The previous SQL is only affected the table feedback. And the feedback is updated regularly. So it won't consume too much resources.

But the fetch all table id from stats_meta will affect all the tables.

If the amount of the table is very large. The feedback may not be too much. But the stats_meta may be very large.

This will become a big performance regression.

Reply:
It is better to get `table_id` first from mysql.stats_feedback, then get feedback? this can get the exact table ids.